manifests/dev/service-binding-operator.v0.0.11-dev.clusterserviceversion.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"apps.openshift.io/v1alpha1","kind":"ServiceBindingRequest","metadata":{"name":"example-servicebindingrequest"},"spec":{"applicationSelector":{"matchLabels":{"connects-to":"postgres","environment":"production"},"resourceKind":"Deployment"},"backingSelector":{"resourceName":"database.example.org","resourceVersion":"v1alpha1"}}}]'
+    capabilities: Basic Install
+    containerImage: REPLACE_IMAGE:REPLACE_VERSION
+    description: An operator to support binding capabilities between imported apps and operator backed services
+  name: service-binding-operator.vREPLACE_VERSION
```

Comment:
"vREPLACE_VERSION" and not "REPLACE_VERSION?"

Reply:
Yes `vREPLACE_VERSION` - name of a CSV is `<name>.v<version>` (such as [`etcdoperator.v0.9.4`](https://github.com/operator-framework/community-operators/blob/master/community-operators/etcd/etcdoperator.v0.9.4.clusterserviceversion.yaml#L6))